### DISCUSSION TITLE
[5.x]: Overview of where and whether fields are used

### DISCUSSION THREAD

**Discussion by handplant** [craftcms/cms#14384] [Ideas] [answered]
It would be advantageous to see if a field is being used in entries, and if so, in which ones. Sometimes, one has field corpses and is hesitant to delete them because they're not entirely certain whether they're truly unused.

**Comment by Mosnar** [comment]
Strong agree here - especially for existing sites migrated to Craft 5. The number of fields can be pretty enormous and cleaning them up is a little scary.

**Comment by brandonkelly** [comment, marked as answer]
Added a “Used by” list to field edit pages, for the next release (#14397) 🎉

**Reply by brandonkelly** [reply]
5.0.0-beta.2 is out now with that change.
